CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
```

Comment:
Avoid leaving TODO comments and temporary fields in committed code. The 'salary' field is redundant, improperly typed (string instead of decimal), and violates naming conventions. It should be removed before merging.

```suggestion
// Removed temporary 'salary' field before merge
```